babel.config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "presets": [
+        "@babel/typescript"
+    ],
+    "plugins": [
+        "@babel/plugin-transform-exponentiation-operator",
+        "@babel/plugin-transform-regenerator",
+        "@babel/plugin-transform-classes",
+        "@babel/plugin-transform-object-super",
+        "@babel/plugin-transform-async-to-generator",
+        "@babel/proposal-class-properties",
+        "@babel/proposal-object-rest-spread",
+        [ "@babel/plugin-proposal-unicode-property-regex",       { useUnicodeFlag: false }]
```

Comment:
This one in particular works around https://github.com/dlclark/regexp2/pull/52 no idea if it breaks something else though